sema/check/service.py

```
@@ -0,0 +1,61 @@
+    for file in yaml_files:
+        log.debug(f"Loading {file}")
+        try:
+            with open(os.path.join(str(input_folder), file), "r") as stream:
+                try:
+                    data = yaml.safe_load(stream)
+                    if data:
+                        rules.extend(data)
+                except yaml.YAMLError as exc:
+                    log.error(f"Error parsing {file}: {exc}")
+        except Exception as e:
+            log.exception(f"Error loading {file}: {e}")
```

Ruff

Logging statement uses f-string

(G004)